cmd/machine-config-daemon/pivot.go

```diff
@@ -231,7 +231,12 @@ func run(_ *cobra.Command, args []string) (retErr error) {
 
 	client := daemon.NewNodeUpdaterClient()
 
-	_, changed, err := client.PullAndRebase(container, keep)
+	var osImageContentDir string
+	var err error
+	if osImageContentDir, err = daemon.ExtractOSImage(container); err != nil {
+		return err
+	}
```

Comment:
any reason not to do:
```go
osImageContentDir, err = daemon.ExtractOSImage(container)
if err != nil {
    return err
}
```

Reply:
Not really, missed to refactor at the end in proper way. fixed

Reply:
This breaks OKD as it doesn't have `oc` in FCOS image. From `release-image.log` on bootstrap node: 
```
Jul 29 08:25:43 ip-10-0-7-95 release-image-download.sh[1450]: I0729 08:25:43.638761    1450 run.go:18] Running: oc image extract --path /:/run/mco-machine-os-content/os-content-779087097 registry.svc.ci.openshift.org/ci-op-lx6rlf8w/stable@sha256:f5283b939e47e2158602d2954f9f1ce1e18ffafb2e711d4187f5954cacc050a6
Jul 29 08:25:43 ip-10-0-7-95 release-image-download.sh[1450]: W0729 08:25:43.638849    1450 run.go:44] oc failed: running oc image extract --path /:/run/mco-machine-os-content/os-content-779087097 registry.svc.ci.openshift.org/ci-op-lx6rlf8w/stable@sha256:f5283b939e47e2158602d2954f9f1ce1e18ffafb2e711d4187f5954cacc050a6 failed: : exec: "oc": executable file not found in $PATH; retrying...
```

Reply:
decided to follow up thanks Vadim :pray: https://github.com/openshift/machine-config-operator/pull/1941#issuecomment-665538414